### DISCUSSION TITLE
RFC: Add pages, or documents, about who is using the project

### DISCUSSION THREAD

**Discussion by ghost** [rbatis/rbatis#270] [General]
rbatis 是一个强大的ORM框架，假以时日，一定发展的很不错。

因此，我自豪的宣布，我开发的开源项目 NFido ， 自豪的使用了rbatis ORM 类库。

不过rbatis现在好像没有这个文档，所以是不是可以加一个专门呈现谁在用rbatis的页面？

**Comment by zhuxiujia** [comment]
> rbatis 是一个强大的ORM框架，假以时日，一定发展的很不错。
> 
> 因此，我自豪的宣布，我开发的开源项目 NFido ， 自豪的使用了rbatis ORM 类库。
> 
> 不过rbatis现在好像没有这个文档，所以是不是可以加一个专门呈现谁在用rbatis的页面？

好主意，请提供项目地址，项目logo，项目介绍。我会在主页面排版谁在使用rbatis

**Comment by ghost** [comment]
项目名称: NFido
项目简介：Rust 开发的 社区论坛系统
项目地址: https://github.com/nfido/nfido.git
项目Logo：

![image](https://user-images.githubusercontent.com/278153/160108965-bdc2e6fd-ce7d-4a80-887f-a4c094e13f6c.png)

**Comment by ghost** [comment]
谢谢，我已经看到了。这个问题可以关闭了，以后一定会有更多的成功案例加入到项目介绍中来，rbatis必定越来越好

**Comment by zhuxiujia** [comment]
项目名称: cassie_axum
项目简介：一个基于rust axum的web 框架 高性能易上手 可用于构建任何web后端项目 供rust爱好者学习使用,基础缓存定义,redis,orm框架选用Ribatis,casbin-rs集成,适配器编写 ,用户权限jwt 融合casbin-rs
项目地址: https://gitee.com/stringlxd/cassie_axum
项目Logo：
![微信图片_20220410205329](https://user-images.githubusercontent.com/3481148/162619158-bef0caaa-501b-4680-87e0-34157620b84c.png)
